### PR TITLE
Feature/refactoring

### PR DESCRIPTION
**Pull request by sfodik**

### CODE REVIEW

File: src/components/card/Card.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    const [modalShow, setModalShow] = useState(false);
+
+    const openModal = () => {
+        setModalShow(true)
+    };
+
+    return (
+        <CardProduct className="sc-card-product">
+            <CardMedia>
+                <Link to="/item-details-01"><img src={data.item.image_preview_url ? data.item.image_preview_url : data.item.img} alt="axies" /></Link>
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: data.item.image_preview_url || data.item.img

Comment (victoriaridna) [inline src/components/card/Card.tsx]: Move "/item-details-01" to the constants

File: src/components/card/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+const Card = data => {
+
+    const [modalShow, setModalShow] = useState(false);
+
+    const openModal = () => {
+        setModalShow(true)
+    };
+
+    return (
+        <CardProduct className="sc-card-product">
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: rewrite style to style component

File: src/components/card/Card.tsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+            <CardMedia>
+                <Link to="/item-details-01"><img src={data.item.image_preview_url ? data.item.image_preview_url : data.item.img} alt="axies" /></Link>
+                <Link to={ROUTES.MOCKED_LOGIN_LINK} className="wishlist-button heart"><span className="number-like">{data.item.wishlist !== undefined ? data.item.wishlist : ''}</span></Link>
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: 1.rewrite styles to styled-component
2.data.item.wishlist || ''

File: src/components/card/Card.tsx [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+                <CardCountdown/>
+                <ButtonPlaceBid className="button-place-bid">
+                    <CardBtn onClick={() => openModal() } className="sc-button style-place-bid style bag fl-button pri-3"><span>Place Bid</span></CardBtn>
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: 1.rewrite styles to styled-component

File: src/components/card/Card.tsx [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+                </ButtonPlaceBid>
+            </CardMedia>
+            <CardTitle className="card-title">
+                <TitleH5><Link to="/item-details-01">"{data.item.name ? data.item.name : data.item.title}"</Link></TitleH5>
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: 1.rewrite styles to styled-component
2.data.item.name || data.item.title
3.Move "/item-details-01" to the constants

File: src/components/card/Card.tsx [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+                <Tags className="tags">{data.item.tags || 'tag'}</Tags>
+            </CardTitle>
+            <MetaInfo className="meta-info">
+                <div className="author">
+                    <Avatar>
+                        <img src={data.item.imgAuthor !== undefined ? data.item.imgAuthor : 'https://pbs.twimg.com/media/FRD-mSRXIAAaOK4.png'} alt="axies" />
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: 1. add constant for mocked link inside this file 'https://pbs.twimg.com/media/FRD-mSRXIAAaOK4.png'
2.data.item.imgAuthor || mockedLink

File: src/components/card/Card.tsx [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+                    </Avatar>
+                    <div className="info">
+                        <span>Creator</span>
+                        <h6> <Link to="/authors-02">{data.item.nameAuthor !== undefined ? data.item.nameAuthor : 'Shiba Satoshi'}
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: 1.data.item.nameAuthor || 'Shiba Satoshi'
2.Move "/authors-02"" to the constants

File: src/components/card/Card.tsx [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+                        </Link> </h6>
+                    </div>
+                </div>
+                <div className="price">
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: rewrite styles to styled-component

File: src/components/card/Card.tsx [hunk 9]
```diff
@@ -0,0 +1,50 @@
...
+                    <span>Current Bid</span>
+                    <h6>{data.item.price !== undefined ? data.item.price : '5 ETH'}</h6>
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: data.item.price || '5 ETH'

File: src/components/card/Card.tsx [hunk 10]
```diff
@@ -0,0 +1,50 @@
...
+            <CardTitle className="card-title">
+                <TitleH5><Link to="/item-details-01">"{data.item.name ? data.item.name : data.item.title}"</Link></TitleH5>
+                <Tags className="tags">{data.item.tags || 'tag'}</Tags>
+            </CardTitle>
+            <MetaInfo className="meta-info">
+                <div className="author">
+                    <Avatar>
+                        <img src={data.item.imgAuthor !== undefined ? data.item.imgAuthor : 'https://pbs.twimg.com/media/FRD-mSRXIAAaOK4.png'} alt="axies" />
+                    </Avatar>
+                    <div className="info">
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: rewrite styles to styled-component

File: src/components/card/Card.tsx [hunk 11]
```diff
@@ -0,0 +1,50 @@
...
+                <ButtonPlaceBid className="button-place-bid">
+                    <CardBtn onClick={() => openModal() } className="sc-button style-place-bid style bag fl-button pri-3"><span>Place Bid</span></CardBtn>
+                </ButtonPlaceBid>
+            </CardMedia>
+            <CardTitle className="card-title">
+                <TitleH5><Link to="/item-details-01">"{data.item.name ? data.item.name : data.item.title}"</Link></TitleH5>
+                <Tags className="tags">{data.item.tags || 'tag'}</Tags>
+            </CardTitle>
+            <MetaInfo className="meta-info">
+                <div className="author">
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: rewrite styles to styled-component

File: src/components/card/Card.tsx [hunk 12]
```diff
@@ -0,0 +1,50 @@
...
+                <CardCountdown/>
+                <ButtonPlaceBid className="button-place-bid">
+                    <CardBtn onClick={() => openModal() } className="sc-button style-place-bid style bag fl-button pri-3"><span>Place Bid</span></CardBtn>
+                </ButtonPlaceBid>
+            </CardMedia>
+            <CardTitle className="card-title">
+                <TitleH5><Link to="/item-details-01">"{data.item.name ? data.item.name : data.item.title}"</Link></TitleH5>
+                <Tags className="tags">{data.item.tags || 'tag'}</Tags>
+            </CardTitle>
+            <MetaInfo className="meta-info">
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: rewrite styles to styled-component

File: src/components/card/Card.tsx [hunk 13]
```diff
@@ -0,0 +1,50 @@
...
+                <Link to="/item-details-01"><img src={data.item.image_preview_url ? data.item.image_preview_url : data.item.img} alt="axies" /></Link>
+                <Link to={ROUTES.MOCKED_LOGIN_LINK} className="wishlist-button heart"><span className="number-like">{data.item.wishlist !== undefined ? data.item.wishlist : ''}</span></Link>
+                <CardCountdown/>
+                <ButtonPlaceBid className="button-place-bid">
+                    <CardBtn onClick={() => openModal() } className="sc-button style-place-bid style bag fl-button pri-3"><span>Place Bid</span></CardBtn>
+                </ButtonPlaceBid>
+            </CardMedia>
+            <CardTitle className="card-title">
+                <TitleH5><Link to="/item-details-01">"{data.item.name ? data.item.name : data.item.title}"</Link></TitleH5>
+                <Tags className="tags">{data.item.tags || 'tag'}</Tags>
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: rewrite styles to styled-component

File: src/components/card/Card.tsx [hunk 14]
```diff
@@ -0,0 +1,50 @@
...
+        setModalShow(true)
+    };
+
+    return (
+        <CardProduct className="sc-card-product">
+            <CardMedia>
+                <Link to="/item-details-01"><img src={data.item.image_preview_url ? data.item.image_preview_url : data.item.img} alt="axies" /></Link>
+                <Link to={ROUTES.MOCKED_LOGIN_LINK} className="wishlist-button heart"><span className="number-like">{data.item.wishlist !== undefined ? data.item.wishlist : ''}</span></Link>
+                <CardCountdown/>
+                <ButtonPlaceBid className="button-place-bid">
```
Comment (victoriaridna) [inline src/components/card/Card.tsx]: rewrite styles to styled-component

**Review by victoriaridna — APPROVED** [review]

File: src/components/card/Card.tsx (hunk 2 above)
Comment (sfodik) [reply src/components/card/Card.tsx]: image
Needed?

**Pull request merged**